### ISSUE TITLE
Docs: Secret "Alternative names" description is unclear

### ISSUE THREAD

**Issue by Nold360** [woodpecker-ci/woodpecker#1445]
### Component

other

### Describe the bug

I think the description for [Secrets - Alternative Names](https://woodpecker-ci.org/docs/next/usage/secrets#alternate-names) is a bit unclear of HOW exactly secrets are included in the pipeline. 

1. Secrets will always become a Environment Variable
2. Environment Variables will always[?] be uppercase(!) 
 
It seems not logical to me that this:
```yaml
secrets:
- source: docker_prod_password
  target: docker_password
```

Becomes `$DOCKER_PASSWORD` as variables can also be lower case in shell.

### System Info

```shell
next
```

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/woodpecker-ci/woodpecker/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://woodpecker-ci.org/docs/intro).
- [X] Check that there isn't [already an issue](https://github.com/woodpecker-ci/woodpecker/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A join our [Discord Chat Server](https://discord.gg/fcMQqSMXJy) or the [Matrix room](https://matrix.to/#/#woodpecker:matrix.org).

**Comment by 6543** [comment]
hmm I'm thinking if we should just drop this syntax and use `from_secret` for all things

**Comment by lafriks** [comment]
Yeah, env definition with from_secret is much easier to use and remember
